.github/workflows/build.yaml

```diff
@@ -13,7 +13,9 @@ on:
       - "!example/**.yaml"
       - "!helm/**"
       - "Makefile"
-      - "!docs/src/**"
+      - "!docs/src/**.md"
+      - "!docs/src/developers/**.md"
+      - "!docs/src/topics/**.md"
```

Comment:
Your previous configuration should work (cf [doc](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths)). You also need to specify these paths in the `pull_request` section. Here it only works for `push`event.